templates/tree.html.erb

```diff
@@ -38,8 +38,14 @@
 </style>
 <body>
   <div id="tree-container"></div>
+<% if options[:local_dependencies]%>
+  <% Dir.entries("js").each do |jsfile|%>
+    <%= "<script>#{File.read File.join('js',jsfile)}</script>" if jsfile != '.' && jsfile != '..' %>
+  <%end%>
+<% else%>
```

Comment:
dirtree works exactly as it is now, no changes to anything except a method that is executed to change the output if the `-l` was passed.
the change is searching for any script tag and replace it with the script tag src content.
that is all.